apply_extra

```diff
@@ -4,4 +4,7 @@ tar -xzvf Stencyl-64-full.tar.gz
 rm -rf Stencyl-64-full.tar.gz
 
 # Export stencyl icon
- install -D /app/extra/data/other/icon-30x30.png /app/extra/export/share/icons/hicolor/64x64/apps/com.stencyl.Game.png
+
+install -D /app/extra/data/other/icon-30x30.png /app/extra/export/share/icons/hicolor/64x64/apps/com.stencyl.Game.png
+install -D /app/temp/com.stencyl.Game.desktop /app/extra/export/share/applications/com.stencyl.Game.desktop
```

Comment:
Maybe you should remove both `/app/temp/com.stencyl.Game.desktop` and `/app/temp/com.stencyl.Game.appdata.xml` after installing them?